test/bpf/map-emulation-demo-test.c

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright (C) 2021 Authors of Cilium */
+
+// source file for map_emulation_demo.h.
+// It contains contains main functions to run test functions in map_emulation_demo.h.
+// It is used to perform unit test on functions in map_emulation_demo.h.
+
+#include "tests/map_emulation_demo_test.h"
```

Comment:
If this is a real test then maybe not calling it "demo"? If it's just a demo then maybe it shouldn't be part of CI?

Reply:
You are right. I just excluded it from CI.